index.js

```diff
@@ -35,22 +30,18 @@ async function setupPlugin({ config, global }) {
 
 
 async function runEveryMinute({ config, global, cache }) {
-    const lastRun = await cache.get('lastRun')
-    if (
-        lastRun &&
-        new Date().getTime() - Number(lastRun) < 3600000 // 60*60*1000ms = 1 hour
-    ) {
-        return
-    }
     let allPostHogAnnotations = []
-    let next = `${global.posthogHost}/api/annotation/?scope=organization&deleted=false`
+    let next = true          
     while (next) {
-        const annotationsResponse = await fetchWithRetry(next, global.posthogOptions)
+        const annotationsResponse = await posthog.api.get(next === true ? '/api/annotation/?scope=organization&deleted=false' : next, {
```

Comment:
@yakkomajuri This is returning 
![image](https://user-images.githubusercontent.com/1727427/153608774-bac4b2f7-11e6-467a-85ae-8d755e6cc1c3.png)
for some reason. Any ideas?